testapps/testapp/build.gradle

```diff
@@ -29,7 +29,7 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
-def msalVersion = "5.+"
```

Comment:
should this be reverted?

Reply:
I don't think so. I see similar changes made in last 2 "Merge release branches to dev" PRs